MixpanelIterableSyncTest/mixpanel_iterable_sync.ipynb

```
@@ -64,805 +78,278 @@
-      "cell_type": "markdown",
-      "metadata": {
-        "id": "Ib-w4ejy1aHn"
-      },
-      "source": [
-        "Home Page Visited\n",
-        "---\n",
-        "---\n",
-        "\n"
+        "import csv, json, uuid\n",
+        "\n",
+        "def send_mixpanel_event(event_name, properties):\n",
```

Ruff (0.12.2)

Probable use of `requests` call without timeout

(S113)